test/stdlib/StringCreate.swift

```diff
@@ -99,8 +99,17 @@ if #available(macOS 10.16, iOS 14.0, watchOS 7.0, tvOS 14.0, *) {
       $0[0] = UInt8(ascii: "4")
       $0[1] = UInt8(ascii: "2")
       $0[2] = UInt8(ascii: "X")
+      ($0.baseAddress! + 2).deinitialize(count: 1)
```

Comment:
Is this required? I am interested in ensuring that our implementation provides reasonable behavior even in the case of a minor API contract violation.

Reply:
Fair enough, I’ll add a note that the API contract violation is deliberate then. My interest is in making sure that we have a similar test case without a contract violation, which deinitializing guarantees.